### ISSUE TITLE
Generating Embeddings

### ISSUE THREAD

**Issue by MarwaEssam** [allenai/longformer#164]
I am trying to run the model and as an initial step, I need to generate embeddings for text, that I can use later for classification. I tried generating the embedding from the CLS vector of the last hidden state of the model. Every time though I get a different embedding vector for the same sentence. Isn't this supposed to be fixed as I am in an evaluation mode.

**Here is my code:**

```
config = LongformerConfig.from_pretrained('longformer-base-4096/')
    config.attention_mode = 'sliding_chunks'
    model = LongformerModel.from_pretrained('longformer-base-4096/', gradient_checkpointing=True)
    model.eval()
    # tokenizer = RobertaTokenizer.from_pretrained('roberta-base')
    tokenizer = LongformerTokenizerFast.from_pretrained('roberta-base')
    tokenizer.model_max_length = 4096
    SAMPLE_TEXT = ' '.join(["What is the capital of Egypt? Cairo is the capital of Egypt."]*1)  # long input document
    tok=tokenizer.encode(text,truncation=True)
    input_ids = torch.tensor(tok).unsqueeze(0)
    attention_mask = torch.ones(input_ids.shape, dtype=torch.long,
                               device=input_ids.device)
   output = model(input_ids, attention_mask=attention_mask,output_hidden_states=True)
  result=output.last_hidden_state.squeeze()[0]
```

I also tried getting it this way: 
`result=output[0].squeeze()[0]`

also using
`result= model.embeddings(input_ids)[0][0]`

still different embeddings each time. Any help?

**Comment by BinchaoPeng** [comment]
I also want to get feature vector through the longformer,so do you make it successfully?could you tell me how to implement the steps? thanks!
